README.adoc

```diff
@@ -54,6 +54,17 @@ POST /authenticate
 }
 ----
 
+----
+400 Bad Request
+----
+[source,json]
+----
+{
+    "code": 1,
+    "message": "Unknown user bob"
```

Comment:
The preferred way of handling this situation is to return something like "your credentials do not match what we have". Being specific about not finding a user or the password does not match allows for fishing and is not good practice.